src/components/HomepageFeatures/index.tsx

```diff
@@ -181,7 +181,7 @@ interface FeatureProps {
 
 function Feature({ title, description }: FeatureProps) {
   return (
-    <div className={clsx("col col--4")}>
+    <div className={clsx("col col--6 margin-bottom--lg")}>
```

Comment:
This needs to be reverted as well, I think.

```suggestion
    <div className={clsx("col col--4")}>
```